CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [#18](https://github.com/dblock/strava-ruby-client/pull/18): Testing against Ruby 2.5.3 and 2.6.0 - [@lucianosousa](https://github.com/lucianosousa).
 * [#18](https://github.com/dblock/strava-ruby-client/pull/18): Upgraded Rubocop to 0.61.1 - [@lucianosousa](https://github.com/lucianosousa).
 * [#21](https://github.com/dblock/strava-ruby-client/pull/21): Include headers in error response - [@jameschevalier](https://github.com/jameschevalier).
+* [#23](https://github.com/dblock/strava-ruby-client/pull/23): Minor: handle special error format for create_upload endpoint. #22 - [@ylecuyer](https://github.com/ylecuyer).
```

Comment:
This is my no means minor. You’ve fixed a bug in which upload errors aren’t properly handled. So just say “Fixed handling of create_upload errors”.

I typically also do [#23](...), [#22](...) and no period in the sentence.